src/main/scala/scatan/model/game/ops/DevCardOps.scala

```diff
@@ -0,0 +1,85 @@
+package scatan.model.game.ops
+
+import scatan.model.components.{DevelopmentCard, ResourceType}
+import scatan.model.game.ScatanState
+import scatan.model.game.config.ScatanPlayer
+import scatan.model.game.ops.AwardOps.awards
+
+object DevCardOps:
```

Comment:
Consider

```suggestion
object DevelopmentCardOps:
```
in order to be esplicative